apps/examples/stdk_smart_switch/Kconfig

```diff
@@ -0,0 +1,11 @@
+#
+# For a description of the syntax of this configuration file,
+# see misc/tools/kconfig-language.txt.
```

Comment:
We don't have this file. Could you change this line as we use?

Reply:
fixed, and also for smart_lamp example, thanks.